python/python.lua

```diff
@@ -0,0 +1,224 @@
+---    Copyright 2024 [lihan aooohan@gmail.com]
+--
+--   Licensed under the Apache License, Version 2.0 (the "License");
+--   you may not use this file except in compliance with the License.
+--   You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+--   Unless required by applicable law or agreed to in writing, software
+--   distributed under the License is distributed on an "AS IS" BASIS,
+--   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+--   See the License for the specific language governing permissions and
+--   limitations under the License.
+
+local http = require("http")
+local html = require("html")
+OS_TYPE = ""
+ARCH_TYPE = ""
+
+local PYTHON_URL = "https://www.python.org/ftp/python/"
+
+local DOWNLOAD_SOURCE = {
+    --- TODO support zip or web-based installers
+    WEB_BASED = "https://www.python.org/ftp/python/%s/python-%s%s-webinstall.exe",
+    ZIP = "https://www.python.org/ftp/python/%s/python-%s-embed-%s.zip",
+    MSI = "",
+    --- Currently only exe installers are supported
+    EXE = "https://www.python.org/ftp/python/%s/python-%s%s.exe",
+    SOURCE = "https://www.python.org/ftp/python/%s/Python-%s.tar.xz"
+}
+
+PLUGIN = {
+    --- Plugin name
+    name = "python",
+    --- Plugin author
+    author = "aooohan",
+    --- Plugin version
+    version = "0.0.1",
+    description = "For Windows, only support >=3.5.0, but no restrictions for unix-like",
+    -- Update URL
+    updateUrl = "https://github.com/aooohan/version-fox-plugins/blob/main/python/python.lua",
+    minRuntimeVersion = "0.2.2",
```

Comment:
Does this mean it will work on version 0.2.2 and later

Reply:
Opps, I forgot to change it here, it should be 0.2.3